fineract-provider/src/main/java/org/apache/fineract/useradministration/service/PasswordPreferencesWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License. You may obtain a copy of the License at
- *
+ * <p>
```

Comment:
This will probably fail. Before you submit please run `./gradlew spotlessApply` to fix (most) of these (formatting) errors.

Reply:
Done. I'll make sure to review git differecnes before opening future PRs to avoid this.